src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -1228,6 +1229,77 @@ void LottieParser::postProcess(Array<LottieGlyph*>& glyphes)
 /* External Class Implementation                                        */
 /************************************************************************/
 
+const char* LottieParser::sid()
+{
+    //verify json.
+    if (!parseNext()) return nullptr;
+    enterObject();
+
+    return nextObjectKey();
+}
+
+
+bool LottieParser::parse(LottieSlot* slot)
+{
+    enterObject();
+
+    // Generate context
+    LottieParser::Context context;
+    this->context = &context;
+
+    // Find target object
+    auto targetObj = comp->find(slot->sid);
+
+    switch (slot->type) {
+        case LottieSlot::Type::ColorStop: {
+            auto gradient = new LottieGradient;
+            this->context->gradient = gradient;
+
+            while (auto key = nextObjectKey()) {
+                if (!strcmp(key, "p")) parseProperty(gradient->colorStops);
+                else skip(key);
+            }
+
+            LottieOverrider::override(targetObj, gradient);
+            delete(gradient);
```

Comment:
assuming this memory deletion also delete the property that has been belonged to the targetObj. So memory corruption... Please double check,